### PR TITLE
cloud: Update monitor-built-in-alerting.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-built-in-alerting.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ If you are a member of a project and you want to get alert notification emails o
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
 2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
 
-    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alerts**.
-    - If you only have one project, click **Admin** > **Alerts**.
+    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alert Subscription**.
+    - If you only have one project, click **Admin** > **Alert Subscription**.
 
-3. Enter your email address, and then click **Subscribe**.
+3. Click **Add Subscriber**, enter your email address in the popup window and then click **Add**.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:26]:
```suggestion
3. Click **Add Subscriber**, enter your email address in the displayed dialog, and then click **Add**.
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 2]
```diff
@@ -20,12 +20,12 @@ If you are a member of a project and you want to get alert notification emails o
...
 
-To minimize the number of alert emails sent to subscribers, TiDB Cloud aggregates alerts into a single email that is sent every 3 hours.
+If the condition triggering an alert does not change, the alert will send email notifications every 3 hours .
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:28]:
```suggestion
If the condition that triggers an alert remains unchanged, the alert sends email notifications every 3 hours.
```

Comment (qiancai) [reply tidb-cloud/monitor-built-in-alerting.md:28]: alert condition 和这篇文档后面的 “TiDB Cloud built-in alert conditions” 也是对应的

File: tidb-cloud/monitor-built-in-alerting.md [hunk 3]
```diff
@@ -34,10 +34,10 @@ If you no longer want to receive alert notification emails of clusters in your p
 1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
 2. In the left navigation pane of the [**Clusters**](https://tidbcloud.com/console/clusters) page, do one of the following:
 
-    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alerts**.
-    - If you only have one project, click **Admin** > **Alerts**.
+    - If you have multiple projects, switch to the target project, and then click **Admin** > **Alert Subscription**.
+    - If you only have one project, click **Admin** > **Alert Subscription**.
 
-3. In the right pane, locate your email address and click **Delete**.
+3. Locate your email address and click **Unsubscribe**.
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:40]:
```suggestion
3. Locate your email address and click **Unsubscribe**.
4. Click **Delete** to confirm the unsubscription.
```

**Review by baurine — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**